### PR TITLE
{devel}[system] Eclipse v4.8 (Photon)

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+versionsuffix = "-parallel-photon-R"  # For scientific computing
+
+homepage = 'http://www.eclipse.org'
+description = """Integrated Development Environment"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'http://www.eclipse.org/downloads/download.php?'
+    'file=/technology/epp/downloads/release/photon/R/'
```
Comment (boegel) [inline easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb]: @SimonPinches This is short enough to fit in < 120 chars on a single ine?

File: easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+]
+sources = [{
+    'filename': '%(name)s-%(version)s%(versionsuffix)s.tar.gz',
+    'download_filename': 'eclipse%(versionsuffix)s-linux-gtk-x86_64.tar.gz'
```
Comment (boegel) [inline easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb]: Wait, they don't provide a properly versioned tarball!?

File: easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb]: Okay - I'll push a fix for this.

File: easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb]: Doesn't seem so :-(

**Comment by boegel** [comment]
It looks like the "source" file is no longer available for download, so I'll close this, there's little point in merging this...

**Pull request closed without merging**